spec/support/vcr.rb

```diff
@@ -8,7 +8,7 @@
     serialize_with: :json,
     preserve_exact_body_bytes:  true,
     decode_compressed_response: true,
-    record: ENV['TRAVIS'] ? :none : :once
+    record: ENV['TRAVIS'] ? :none : :new_episodes
```

Comment:
Do we actually want to change this VCR setting permanently?

Using `:new_episodes` means that we can inadvertently introduce new requests without getting a error/warning: https://relishapp.com/vcr/vcr/v/2-8-0/docs/record-modes/new-episodes